pkgs/tools/networking/slack-cli/default.nix

```diff
@@ -0,0 +1,27 @@
+# slack-cli must be configured using the SLACK_CLI_TOKEN environment
+# variable. Using `slack init` will not work because it tries to write
+# to the Nix store.
+
+{ stdenv, lib, fetchFromGitHub, makeWrapper, curl, jq }:
+
+stdenv.mkDerivation rec {
+  name = "slack-cli";
```

Comment:
It is customary to add the version to the name: `name = "slack-cli-${version}"`.